crates/lib/src/install/completion.rs

```diff
@@ -275,6 +276,20 @@ pub(crate) async fn impl_completion(
     sysroot: &ostree::Sysroot,
     stateroot: Option<&str>,
 ) -> Result<()> {
+    // Log the completion operation to systemd journal
+    const COMPLETION_JOURNAL_ID: &str = "0f9a8b7c6d5e4f3a2b1c0d9e8f7a6b5c4";
+    let msg = "Starting bootc installation completion";
+    crate::journal::journal_send(
+        Priority::Info,
+        msg,
+        [
+            ("MESSAGE_ID", COMPLETION_JOURNAL_ID),
+            ("BOOTC_OPERATION", "install_completion"),
```

Comment:
This `BOOTC_OPERATION` bit feels slightly ad-hoc; it has a pretty high overlap with the `CODE_FUNC` emitted by the default C library - but that one isn't 

I think I'd vote to drop it for now, and actually fix things so that we have a macro that injects `CODE_FUNC`.

Reply:
Dropped for now (in other files too).